File: Atividades A1/Exercicio 2/GO/TresVetoresEx2/main.go
package main

import "fmt"

func main() {
	// Declarando vetores
	var vetorA, vetorB, vetorC [10]int

	// Recebendo os dados do vetor A
	fmt.Println("Digite os elementos do vetor A:")
	for i := 0; i < 10; i++ {
		fmt.Printf("Elemento %d do vetor A: ", i)
		fmt.Scan(&vetorA[i])
	}

	// Recebendo os dados do vetor B
	fmt.Println("Digite os elementos do vetor B:")
	for i := 0; i < 10; i++ {
		fmt.Printf("Elemento %d do vetor B: ", i)
		fmt.Scan(&vetorB[i])
	}

	// Calculando a diferença entre os vetores A e B e armazenando no vetor C
	for i := 0; i < 10; i++ {
		vetorC[i] = vetorA[i] - vetorB[i]
	}

	// Exibindo os valores do vetor C
	fmt.Println("Vetor C (A - B):")
	for i := 0; i < 10; i++ {
		fmt.Printf("Elemento %d do vetor C: %d\n", i, vetorC[i])
	}
}

File: Atividades A1/Exercicio 2/Java/TresVetoresEx2/src/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        // Declarar vetores
        int[] vetor_a = new int[10];
        int[] vetor_b = new int[10];
        int[] vetor_c = new int[10];


        for (int i = 0; i< 10;i++){
            // Recebe valores dos vetores
            System.out.printf("Digite o elemento %d do vetor A: ", i);
            vetor_a[i] = scanner.nextInt();
            System.out.printf("Digite o elemento %d do vetor B: ", i);
            vetor_b[i] = scanner.nextInt();

            // Calcula a diferença e coloca dentro do vetor C
            vetor_c[i] = vetor_a[i] - vetor_b[i];
        }
        System.out.println("Vetor C: ");
        for (int i = 0; i<10;i++){
            System.out.printf("Elemento %d do vetor C: %d\n", i, vetor_c[i]);
        }
    }
}